kitsune/users/jinja2/users/profile.html

```diff
@@ -11,15 +11,14 @@
 {% block content %}
   <div class="grid_9">
     <article id="profile" class="cf">
-      <div class="avatar">
-        <img src="{{ profile_avatar(profile.user) }}" alt="">
-      </div>
-      {% if user.has_perm('users.change_profile') %}
-        <div id="admin-actions">
-          <a class="edit" href="{{ url('users.edit_profile', profile.user.username) }}">
-            {{ _('Edit user profile') }}
-          </a>
-        </div>
+      {% if profile and profile.is_fxa_migrated %}
```

Comment:
Yes. This should have been:
```
{% if is_owner and profile.is_fxa_migrated %}
```
Because only the owner should see the "account" section

Reply:
Isn't `profile` always present here?